Sources/DefaultsController.swift

```diff
@@ -0,0 +1,65 @@
+import Foundation
+
+enum UserDefaultsState: Equatable {
```

Comment:
Can we change this name? Doesn't seem like a `State` really, seems more like a `StorageSharingPolicy` to me? I'd also rename the `UserDefaults` part to `Storage` to not leak the implementation of it, just in case we decide to change this in the future.